development/docker-compose.yml

```diff
@@ -48,13 +61,13 @@ services:
     tty: true
   celery_beat:
     image: "networktocode/nautobot-dev-py${PYTHON_VER}:local"
-    entrypoint: "nautobot-server celery beat -l INFO -s /tmp/celerybeat-schedule"
```

Comment:
IIRC the reason for `-s /tmp/celerybeat-schedule` was so that a `celerybeat-schedule` file wouldn't be created in the volume-mounted source directory. Is that no longer a concern, or what's the reason for this change?